Automattic/wp-calypso
# Add first post guided tour

This PR adds a guided tour triggered from the onboarding checklist.

# Testing

* Visit `/checklist/<domain>` for your site
* Click 'Do It' for the first post task (whether completed or not - the tour should show)
* Verify that the following steps appear in the tour:

![first-post](https://user-images.githubusercontent.com/1926/34146670-a3da7912-e4e6-11e7-9d49-36b0f30738c5.gif)
